src/main/java/org/broadinstitute/hellbender/utils/read/FlowBasedRead.java

```diff
@@ -820,9 +869,9 @@ private void applyFilteringFlowMatrix(){
     private void clipProbs() {
         for ( int i = 0 ; i < getMaxHmer(); i++ ) {
             for ( int j =0; j < getNFlows(); j++) {
-                if ((flowMatrix[i][j] <= fbargs.probabilityRatioThreshold) &&
+                if ((flowMatrix[i][j] <= perHmerMinErrorProb*3) &&
```

Comment:
where did this `*3` come from? Why is it needed here?

Reply:
@ilyasoifer

Reply:
Clarified the function